src/ui/common/src/components/notifications/NotificationListItem.tsx

```diff
@@ -105,43 +110,21 @@ export const NotificationListItem: React.FC<Props> = ({
         />
       </Box>
 
-      <Typography
-        variant="h6"
-        gutterBottom
-        sx={{
-          fontFamily: 'Monospace',
-          '&:hover': { textDecoration: 'underline' },
-        }}
-      >
-        {notification.content}
-      </Typography>
-
       <Box
         sx={{
           display: 'flex',
-          flexDirection: 'row',
-          justifyContent: 'space-between',
+          justifyContent: 'flex-end',
         }}
       >
-        <Box>
-          <Typography
-            variant="body1"
-            sx={{ fontWeight: 'medium', color: 'gray600' }}
-          >
-            {/* Show notification associated with 'workflow_run' as 'workflow' category */}
-            {association.object === NotificationAssociation.WorkflowDagResult
-              ? NotificationAssociation.Workflow
-              : association.object}
-          </Typography>
-        </Box>
-        <Box>
-          <Typography
-            variant="body1"
-            sx={{ fontWeight: 'light', color: 'gray600' }}
-          >
-            {`${dateString(notification.createdAt)}`}
-          </Typography>
-        </Box>
+        <Typography
+          variant="body1"
+          sx={{
+            fontWeight: 'light',
```

Comment:
Can you post a screenshot of light vs medium font weight please? Curious what the two look like.

Reply:
<img width="1165" alt="image" src="https://user-images.githubusercontent.com/61630725/172474006-1ebfb2be-0f21-42f7-a6e2-1f85cba7fa2d.png">